src/components/ResultDisplay.tsx

```diff
@@ -52,7 +55,7 @@ export function ResultDisplay({ result }: ResultDisplayProps) {
 									: b.estimatedMoney - a.estimatedMoney
 								: a.lifted
 								? 1
-								: -1,
```

Comment:
I think it's this comma deletion that's messing with Prettier. If you run `pnpm run format --write` locally, it should fix it. You might have to `pnpm install` first if your local installed modules are out of date.

Reply:
That was it ty!